src/lib/containers/row_renderers/utils/Icon.tsx

```diff
@@ -140,15 +143,22 @@ const Icon: React.FunctionComponent<IconProps> = ({
   // or if the type of card has a corresponding icon, e.g. 'Publication'
   const Icon = iconSet[value] || iconSet[type]
   // TODO: get rid of dataset icon class, none of the icons should be special cased
-  const datasetIconClass =
-    value === DATASET || type === DATASET ? 'SRC-datasetIcon' : ''
-  if (typeof Icon == 'function') {
-    return (
-      <span className={cssClass}>
-        <Icon />
-      </span>
-    )
+  const datasetCustomStyle =
+    value === DATASET || type === DATASET ? { height: '55px' } : {}
+
+  const className = `iconImg ${cssClass ?? ''}`
+
+  if (Icon == null) {
+    console.warn('Icon type not found:', type)
+    return <></>
+  }
+  if (typeof Icon === 'string') {
+    return <img src={Icon} className={className} />
   }
-  return <img src={Icon} className={`iconImg ${datasetIconClass}`} />
+  return (
+    <span>
+      <Icon className={className} style={datasetCustomStyle} />
```

Comment:
React can call createElement on the item regardless of it's an object or a function. I would bet when our build tools changed, the previous dependency created function components, while the new one creates class components.